src/design-system-team.md.njk

```diff
@@ -1,41 +1,48 @@
 ---
-title: GOV.UK Design System team
-description: about the GOV.UK Design System team and who works on it
+title: GOV.UK Design System and Prototype Kit team
+description: about the GOV.UK Design System and Prototype Kit team and who works on them
 layout: layout-single-page-prose.njk
 ---
 
-# GOV.UK Design System team
+# GOV.UK Design System and Prototype Kit team
 
-The GOV.UK Design System team at the
-[Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) maintains this design system.
+The GOV.UK Design System and Prototype Kit team at the
+[Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) maintains this design system. We also maintain the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/docs).
 
 If you want to contact the team you can
 [get in touch via email or Slack](/get-in-touch/).
 
-## Members
+## GOV.UK Design System team
+- Kelly Lee – Delivery Manager
+- Trang Erskine – Senior Product Manager
+
+### Design System squad
 
-- Amelia Phillips – User Researcher
-- Ben Surgison – Senior Node JS Developer
-- Brett Kyle – Frontend Developer
 - Calvin Lau – Senior Content Designer
 - Charlotte Downs – Interaction Designer
 - Chris Ballantine-Thomas – Senior Interaction Designer
 - Ciandelle Scollan – Interaction Designer
-- Claire Ashworth – Technical Writer
-- Colin Rotherham – Senior Frontend Developer
 - David Cox – Senior Accessibility Specialist
 - Imran Hussain – Community Manager
-- Izabela Podralska – Delivery Manager
-- Joe Lanman – Lead Interaction Designer
 - Katrina Birch – User Researcher
-- Kelly Lee – Delivery Manager
 - Kim Grey – Frontend Developer
```

Comment:
```suggestion
- Kim 'beeps' Grey – Frontend Developer
```

A little self-indulgent, but given beeps is the name I prefer to be called in casual contexts, I'd like if the page included it!

Reply:
Of course!